app/reflexes/admin/orders_reflex.rb

```diff
@@ -1,97 +1,101 @@
 # frozen_string_literal: true
 
-class Admin::OrdersReflex < ApplicationReflex
-  before_reflex :authorize_order, only: [:capture, :ship]
-
-  def capture
-    payment_capture = OrderCaptureService.new(@order)
-
-    if payment_capture.call
-      morph dom_id(@order), render(partial: "spree/admin/orders/table_row",
-                                   locals: { order: @order.reload, success: true })
-    else
-      flash[:error] = with_locale{ payment_capture.gateway_error || I18n.t(:payment_processing_failed) }
-      morph_admin_flashes
+module Admin
+  class OrdersReflex < ApplicationReflex
```

Comment:
Hmmm is rubocop configured for an old version of Ruby/Rails or something? I have a sneaking suspicion this "Class/Module on separate lines" rule was previously recommended for purely technical reasons related to class loading but I don't think it's still needed. In this case I feel like doubling up on indentation throughout the whole file is not actually preferable...?

Reply:
> is rubocop configured for an old version of Ruby/Rails or something?

No, it auto-detects the version.

> I have a sneaking suspicion this "Class/Module on separate lines" rule was previously recommended for purely technical reasons related to class loading but I don't think it's still needed.

I thought that, too. I thought that Zeitwerk made this obsolete. But then I had a problem recently where a class wasn't found. Now checking again, the problem looks like this:

```rb
module OFN
  class User
  end
end

class OFN::Login
  def call
    User.new
  end
end

OFN::Login.new.call
(irb):8:in `call': uninitialized constant OFN::Login::User (NameError)
Did you mean?  OFN::User

# But this works:
module OFN
  class Login
    def call
      User.new
    end
  end
end

OFN::Login.new.call
=> #<OFN::User:0x00007f4b0166b388>
```

Reply:
Ah I see. If you did this then it would find it:
```ruby
class OFN::Login
  def call
    OFN::User.new
  end
end
```
The `OFN::Login` class itself is still defined correctly and can be invoked by other classes without problems. Dropping the `module` just means referencing other classes in the same namespace doesn't automatically add the module name :+1:

Reply:
Yes, so you could use either style on a case-by-case basis but then we don't have a consistent style and some people  may stumble over this namespace issue without knowing why.